namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-cla-backend-uat/resources/s3.tf

```diff
@@ -25,7 +25,9 @@ module "cla_backend_private_reports_bucket" {
       "s3:ListBucket"
     ],
     "Resource": [
-      "$${bucket_arn}"
+      "$${bucket_arn}",
+      "$${module.cla_backend_static_files_bucket.bucket_arn}",
```

Comment:
Remove the double `$$` **only** for module stanzas.

Reply:
Just did a plan `terraform plan` (`Plan: 0 to add, 1 to change, 0 to destroy.`), this is my diff

```
diff --git a/namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-cla-backend-uat/resources/s3.tf b/namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-cla-backend-uat/resources/s3.tf
index 3ef35d68..981ebdba 100644
--- a/namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-cla-backend-uat/resources/s3.tf
+++ b/namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-cla-backend-uat/resources/s3.tf
@@ -26,8 +26,8 @@ module "cla_backend_private_reports_bucket" {
     ],
     "Resource": [
       "$${bucket_arn}",
-      "$${module.cla_backend_static_files_bucket.bucket_arn}",
-      "$${module.cla_backend_deleted_objects_bucket.bucket_arn}"
+      "${module.cla_backend_static_files_bucket.bucket_arn}",
+      "${module.cla_backend_deleted_objects_bucket.bucket_arn}"
     ]
   },
   {
@@ -37,9 +37,9 @@ module "cla_backend_private_reports_bucket" {
       "s3:*"
     ],
     "Resource": [
-      "$${bucket_arn}/*"
-      "$${module.cla_backend_static_files_bucket.bucket_arn}/*",
-      "$${module.cla_backend_deleted_objects_bucket.bucket_arn}/*"
+      "$${bucket_arn}/*",
+      "${module.cla_backend_static_files_bucket.bucket_arn}/*",
+      "${module.cla_backend_deleted_objects_bucket.bucket_arn}/*"
     ]
   }
 ]

```

Reply:
> Remove the double $$ only for module stanzas.

So it would just be `"{module.cla_backend_static_files_bucket.bucket_arn}"`

Reply:
Ah ok, got you. Ignore my last comment